CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_intercom_source v0.6.1
+## ⭐ Features ⭐
+- Added passthrough custom column capability for models depending on source table `conversation_history`. ([#22](https://github.com/fivetran/dbt_intercom_source/pull/22))
```

Comment:
You actually ended up updating the entire passthrough functionality to be way more robust by using the macros from fivetran_utils! Therefore, customers can now take advantage of the arguments: name, alias, transform_sql.

I think it would be beneficial to highlight these changes while also letting customers know their existing passthrough columns will still work as expected.

Reply:
Done.